src/main/java/no/unit/nva/doi/transformer/utils/CrossrefType.java

```diff
@@ -0,0 +1,46 @@
+package no.unit.nva.doi.transformer.utils;
+
+import no.unit.nva.model.PublicationType;
+import nva.commons.utils.SingletonCollector;
+
+import java.util.Arrays;
+
+import static java.util.Objects.isNull;
+
+public enum CrossrefType {
```

Comment:
CrossRefPulibcationType?

Reply:
We should consider this change for everything — it is consistent with the others right now.